content/java/concepts/array-list/terms/removeAll/removeAll.md

```diff
@@ -0,0 +1,68 @@
+---
+Title: '.removeAll()'
+Description: 'Removes multiple elements from an ArrayList.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Data Types'
+  - 'Java'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.removeAll()`** method is used for removing multiple elements from instances of the `ArrayList` class.
+
+## Syntax
+
+A collection of elements, specified by `Collection c`, can be removed from an `ArrayList` instance using the `.removeAll()` method:
```

Comment:
Remove this line and add the description of the parameters `ArrayList2` and `ArrayList1` below the syntax in this format - 
```
- `parameterName`: Parameter description.
- `parameterName`: Parameter description.
```